src/main/java/seedu/address/logic/commands/StatisticsCommand.java

```diff
@@ -23,12 +24,13 @@ public class StatisticsCommand extends Command {
             + ": Displays the general statistics regarding the current people list in SocialBook.\n"
             + "Example: " + COMMAND_WORD;
     public static final String MESSAGE_DISPLAY_STATISTICS_SUCCESS = "Here are all the statistics:\n%s";
-    public static final String MESSAGE_DISPLAY_TOTAL_PEOPLE = "Total # Of People: %s";
-    public static final String MESSAGE_DISPLAY_HIGH_PRIORITY = "# Of HIGH Priority People: %s";
-    public static final String MESSAGE_DISPLAY_MEDIUM_PRIORITY = "# Of MEDIUM Priority People: %s";
-    public static final String MESSAGE_DISPLAY_LOW_PRIORITY = "# Of LOW Priority People: %s";
-    public static final String MESSAGE_DISPLAY_LOW_INCOME = "# Of People Income <= 800: %s";
-    public static final String MESSAGE_DISPLAY_APPOINTMENTS_SOON = "# Of Appointments In Next 7 Days: %s";
+    public static final String MESSAGE_DISPLAY_TOTAL_PEOPLE = "Total Number Of People: %s";
+    public static final String MESSAGE_DISPLAY_HIGH_PRIORITY = "Number Of HIGH Priority People: %s";
+    public static final String MESSAGE_DISPLAY_MEDIUM_PRIORITY = "Number Of MEDIUM Priority People: %s";
+    public static final String MESSAGE_DISPLAY_LOW_PRIORITY = "Number Of LOW Priority People: %s";
+    public static final String MESSAGE_DISPLAY_LOW_INCOME = "Number Of People Household Income <= 4000: %s";
```

Comment:
Just a minor grammatical suggestion - perhaps could be phrased as "Number of People with Household Income ..."

Reply:
thnks have made the change. btw is there any other general stats that u think would be useful to include as well?